cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -257,9 +257,18 @@ func (o *Options) config(kubeAPIServerConfig *rest.Config, kubeClient *kubernete
 		}, nil
 	}
 
+	gardenerKubeClient, err := kubernetes.NewForConfig(gardenerAPIServerConfig.ClientConfig)
+	if err != nil {
+		return nil, err
+	}
+	gardenerDynamicClient, err := dynamic.NewForConfig(gardenerAPIServerConfig.ClientConfig)
```

Comment:
Just "dynamicClient"?

Reply:
Here client is created from `gardenerAPIServerConfig.ClientConfig` that's why I named it `gardenerDynamicClient `.